modules/get-started/pages/whats-new.adoc

```diff
@@ -1,79 +1,95 @@
+== Data transforms
 
-== Server-side schema ID validation
+Redpanda now provides the framework to build and deploy inline transformations (xref:develop:data-transforms/index.adoc[data transforms]) on data written to Redpanda topics, delivering processed and validated data to consumers in the format they expect. For example, you may have consumers that require you to redact credit card numbers or convert JSON to Avro. Data transforms use WebAssembly (Wasm) engines inside a broker, which eliminates the need to manage a separate stream processing environment or use third-party tools.
 
-With this enterprise feature, records produced to a topic may use a serializer/deserializer client library, such as Confluent's SerDes library, to encode their keys and values according to a schema in Schema Registry. xref:manage:schema-id-validation.adoc[Schema ID validation] enables brokers to detect and drop records that don't match the configured schema, as identified by the schema ID. Records associated with unregistered or incorrect schemas are thus detected and dropped earlier, by a broker rather than a downstream consumer.
+== OIDC authentication
 
-== Tiered Storage enhancements
+You can xref:manage:security/authentication.adoc#enable-oidc[enable OIDC] for Redpanda to delegate the authentication process to an external identity provider (IdP) such as Okta, Azure AD, or on-premise Active Directory Federation Service (AD FS). When you enable OIDC, Redpanda does not need to manage user credentials directly, but can instead rely on the trusted authentication capabilities of IdPs. Redpanda's implementation of OIDC provides SASL/OAUTHBEARER support for the Kafka API, and enables standard OIDC authentication across the Schema Registry, HTTP Proxy, and Admin APIs.
 
-As data in object storage grows, the metadata for it grows. To support efficient long-term data retention, Redpanda splits the metadata in object storage, maintaining metadata of only recently-updated segments in memory or local disk, while safely archiving the remaining metadata in object storage and caching it locally on disk. Archived metadata is then loaded only when historical data is accessed. This allows Tiered Storage to handle partitions of virtually any size or retention length.
+== Audit logging 
 
-- xref:manage:tiered-storage.adoc#caching[Fine-grained caching]: To support more concurrent consumers of historical data with less local storage, Redpanda can download small chunks of remote segments to the cache directory. For example, when a client fetch request spans a subsection of a 1 GiB segment, instead of downloading the entire 1 GiB segment, Redpanda can download 16 MiB chunks that contain just enough data required to fulfill the fetch request.
+Redpanda's xref:manage:audit-logging.adoc[audit logging] supports fine-grained reporting of events in the Kafka, HTTP Proxy, Schema Registry, and Admin APIs. Events include:
```

Comment:
"reporting of calls to the Kafka ... "

the events are the audit events. what they're auding is calls to the APIs